src/data/polynomial/degree/definitions.lean

```diff
@@ -736,6 +736,37 @@ begin
   refine add_le_add _ _; apply degree_le_nat_degree,
 end
 
+lemma nat_degree_pow_le {p : polynomial R} {n : ℕ} : (p ^ n).nat_degree ≤ n * p.nat_degree :=
+begin
+  induction n with i hi,
+  { simp },
+  { rw [pow_succ, nat.succ_mul, add_comm],
+    apply le_trans nat_degree_mul_le,
+    exact add_le_add_left hi _ }
+end
+
+@[simp] lemma coeff_pow_degree_mul_degree (p : polynomial R) (n : ℕ) :
```

Comment:
```suggestion
@[simp] lemma coeff_pow_mul_nat_degree (p : polynomial R) (n : ℕ) :
```
the lemma only mentions `degree` once, so why have it twice in the name?

Reply:
That was to make the naming consistent with `coeff_mul_degree_add_degree`

Reply:
Oh, I misinterpreted the naming convention. I'll follow your suggestion then